components/supervisor/cmd/init.go

```diff
@@ -114,18 +114,22 @@ var initCmd = &cobra.Command{
 			}
 		}()
 		// start the reaper to clean up zombie processes
+		reaperChan := make(chan reaper.Status, 10)
```

Comment:
As the Go routine is just checking the PID for equality, and only forwards to `handledByReaper` in a single case, I think the small buffer of 10 is not leading to any blockages here. :+1: